src/main/scala/cognite/spark/v1/StringDataPointsRelation.scala

```diff
@@ -27,8 +29,41 @@ case class StringDataPointsFilter(
     externalId: Option[String]
 )
 
+case class StringDataPointsDelete(
+    id: Option[Long],
+    externalId: Option[String],
+    inclusiveStart: Instant,
+    exclusiveEnd: Instant
+)
+
 class StringDataPointsRelationV1(config: RelationConfig)(override val sqlContext: SQLContext)
-    extends DataPointsRelationV1[StringDataPointsItem](config)(sqlContext) {
+    extends DataPointsRelationV1[StringDataPointsItem](config)(sqlContext)
+    with WritableRelation {
+
+  override def insert(rows: Seq[Row]): IO[Unit] = insertSeqOfRows(rows)
+
+  override def upsert(rows: Seq[Row]): IO[Unit] = insert(rows)
+
+  override def update(rows: Seq[Row]): IO[Unit] = insert(rows)
```

Comment:
Please throw an error on `insert` and `update` that says they're not supported for datapoints, and advise using upsert instead. The way we describe these in the Readme are that insert will stop upon finding a match, while update will stop when not finding a match, and I'd prefer making it clear that we don't do that for datapoints.